CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 8.0.0 (unreleased)
+ 
+- Add EdDSA algorithm
+- `sign`/`verify` now takes a `&[u8]` instead of `&str` to be more flexible
+- `DecodingKey` now own its data
+- Remove deprecated `dangerous_unsafe_decode`
```

Comment:
Hello! I had a quick question: were the other, non-deprecated `dangerous_...` functions supposed to be removed from the API?

Reply:
Yes, you can call https://github.com/Keats/jsonwebtoken/blob/next/src/validation.rs#L112-L114 to your Validation struct instead to achieve the same as the other `dangerous` fns

Reply:
Awesome, thanks!